Game/Resources/elevatorScene/Scripts/elevator_backDoorCon.as

```diff
@@ -61,6 +60,7 @@ class elevator_backDoorCon {
     }
 
     void CloseDoor() {
+        openingDoor = true;
```

Comment:
Is this an oversight? It seems wrong to set `openingDoor` to true when calling `CloseDoor`.

Reply:
umm